x-pack/plugins/apm/server/lib/transactions/get_throughput_charts/transform.ts

```diff
@@ -7,18 +7,21 @@
 import { sortBy } from 'lodash';
 import { NOT_AVAILABLE_LABEL } from '../../../../common/i18n';
 import { ThroughputChartsResponse } from '.';
+import { calculateThroughput } from '../../helpers/calculate_throughput';
+import { SetupTimeRange } from '../../helpers/setup_request';
 
 type ThroughputResultBuckets = Required<ThroughputChartsResponse>['aggregations']['throughput']['buckets'];
 
 export function getThroughputBuckets({
   throughputResultBuckets = [],
   bucketSize,
-  durationAsMinutes,
+  setupTimeRange,
 }: {
   throughputResultBuckets?: ThroughputResultBuckets;
   bucketSize: number;
-  durationAsMinutes: number;
+  setupTimeRange: SetupTimeRange;
 }) {
+  const { start, end } = setupTimeRange;
```

Comment:
It looks like we could also update the calculation below (on line 31):

```
y: bucket.count.value / (bucketSize / 60),
```

Reply:
Btw. this is a good candidate for using the [rate agg](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/search-aggregations-metrics-rate-aggregation.html) that we also have an [issue](https://github.com/elastic/kibana/issues/75885) for